app/pages/Accounts/AccountDetailsPage/AccountDetailsPage.tsx

```diff
@@ -53,7 +57,8 @@ export default withAccountSync(function DetailsPage() {
 
     const isBaker = Boolean(accountInfo?.accountBaker);
     const canTransfer = hasCredentials && Boolean(accountInfo);
-    const accountChanged = !useIsSubsequentRender();
+    const isFirstRender = !useIsSubsequentRender();
+    const accountHasChanged = isFirstRender && !hasLocationState;
```

Comment:
This change does so we stay in the build schedule view, when changing account, which I thought we didn't want.

Reply:
Right you are.. sorry! Fixed in newest commit